CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# UNRELEASED
+# matrix-sdk-crypto-wasm v4.8.0
 
```

Comment:
We normally leave the "UNRELEASED" title at the top, above title for this release.

Reply:
fixed